Instruments/microwave_tuning_gui.py

```diff
@@ -81,8 +82,8 @@ def on_about(self):
         QMessageBox.about(self, "About the demo", msg.strip())
     
     def orig_zoom_limits(self):
-        for ini_val, w in [('9700000',self.textbox1),
-                ('9900000',self.textbox2)]:
+        for ini_val, w in [('9815000',self.textbox1),
+                ('9826000',self.textbox2)]:
```

Comment:
we may want to eventually widen but if it is too wide the points jump over the dip and it errors out